src/elements/fields/CheckboxGroupField.jsx

```diff
@@ -1,6 +1,15 @@
-import React from 'react';
+import React, { useMemo } from 'react';
 import ReactForm from 'react-bootstrap/Form';
 import { bootstrapStyles } from '../styles';
+import { applyCheckboxStyles, composeCheckboxStyle } from './CheckboxField';
+
+const applyCheckboxGroupStyles = (element, applyStyles) => {
+  applyStyles.addTargets(['checkboxGroup']);
```

Comment:
nit: should this be called checkbox rather than checkbox group since it's being applied individually to checkboxes

Reply:
also this style logic should also ideally be in https://github.com/feathery-org/feathery-react/blob/master/src/elements/fields/index.jsx#L30

Reply:
It needs a target other than checkbox. Checkbox refers to the literal input HTML element here. Where as checkboxGroup is the checkbox + label grouping. The checkbox input needs to have a separate width and height than the containing div that holds both the checkbox and the label. Does that make sense?

Reply:
cool make ssense